### ISSUE TITLE
Let's get ready for the sprint

### ISSUE THREAD

**Issue by chadsansing** [MozillaFoundation/mpa-cryptomancer-challenge#15]
Let's get ready for the sprint!

The Global Sprint is almost here! Tomorrow and Friday (and beyond), we'll join together to contribute assets, ideas, and questions to game-based projects that help us teach about online safety in fun, low-risk ways.

Your project leaders will be online from 9 AM to 5 PM, their local time, to answer your questions and help you participate in the sprint. Please don't hesitate to let us know if you need something from us.

Remember to abide by the [Code of Conduct](https://github.com/MozillaFoundation/mpa-cryptomancer-challenge/blob/master/code-of-conduct.md) as we work together. This is a safe space for everyone, and it's our job to maintain it as such.

Please also remember that the big idea is to teach real-life privacy and security habits through the games we adapt, expand, and invent during the sprint. Always ask yourself:

- Does this make sense?
- Does this connect back to real-world habits people can follow?
- Does this seem friendly or inviting to a beginner gamer or web user?

Put your users first.

Here is [a blog post you can share](https://medium.com/@chadsansing/game-time-at-the-global-sprint-a907bdb11b7b) with a last-minute invite to the sprint.

We look forward to working with you and we cannot wait to see what you make of the 2017 Global Sprint!

Please ask any last-minute questions below.

cc @cryptomancer-actual, @tromand, @anterobot, @gael-ian, @Salindurthas, @DMRaven, @dexterduck, @Masterwolf2050, @ashgamedesign, @villain4hire, @Plotanical, @KadeMorton

**Comment by tromand** [comment]
Up and ready :)

**Comment by cryptomancer-actual** [comment]
I'm here, too. I'll be working on sprint related stuff all day.

**Comment by chadsansing** [comment]
Ready to go!

**Comment by cryptomancer-actual** [comment]
Welcome @tromand !

I was wondering if you had any adventures from your own game that we could port over to Cryptomancer just by adding some fantasy elements?

**Comment by cryptomancer-actual** [comment]
Cover image for our project over in Issue #7

**Comment by tromand** [comment]
@cryptomancer-actual hum, hard to tell, the adventure in my game are completely invented by the players in a interactive way. One idea that I see could work is one of my trainee group did try a kind of DDoS on the supervision : the object they were tying to still had a GPS tracker on it, they copied the signal and flushed multiple gps tracker on the canalisation (via WC ^^) to confused the defender team. Maybe we could apply that with shards on megaphid ?

**Comment by cryptomancer-actual** [comment]
@tromand That's a really interesting concept. I could see a "guerilla communications" section developing here... how to protect the geolocation of shardnets through deception/denial techniques. Like you mention, if someone was tracking your location via the "Tracer" spell (which geolocates shards), you could put your shard on a wagon, or on a gigaphid (giant flying insect) and have it run/fly off. Now your adversary is on a wild goose chase. Another technique might involve using scryer-nets (which are like CCTV systems using shards) as a method to communicate with people "off the grid." There might be scryer-nets set up all over a city under marshal law, and being monitored by the authorities. A character could climb up and touch one of these and leave an encrypted message on that network (they'd have to use a strong keyphrase, otherwise the people monitoring the scryer-network can see the message). Then other characters can climb up, touch any of the scryer-net shards, and receive the message (if they shared the keyphrase offline).

**Comment by tromand** [comment]
This goes well for my first message, I messed up the gigaphid name :D 

You could also put multiple shards of the same shardnet on gigaphid (or other animals) and just use a few for actually communicating. Like you will need a lot of tracer spells to follow shards for maybe nothing.

By the way, could we put a golem on a GigaPhid ?

**Comment by cryptomancer-actual** [comment]
Mobile Golem is great/fun idea.

In the game text, it says Golems are pretty big machines... giant steam-powered squid things that take a lot of energy (coal, wind, waterwheel, etc.) to power. I made them require energy, instead of just being magically powered, because I wanted to emphasize critical infrastructure (e.g. electricity) as an attack vector (i.e. destroy the windmill powering the golem and you've basically pulled the plug on a target's network). I also wanted to emphasize opportunities for BCP/DR (business continuity planning / disaster recovery, for those not in IT careers). A secure organization will have a *plan* for when an adversary takes out their golem, etc.

All that being said, fantasy settings contain *magic*, which means you can just hand-wave any idea and just say "this works because special magic." So yeah, a golem on a gigaphid... or just a flying golem that rides along the jet stream would be perfectly OK. And now that I think about it, a flying golem is amazing.

**Comment by tromand** [comment]
Maybe we could "invent" a flying "mini" Golem, with less-capacity than a real one (because of the lack of infra.), maybe can connect to less shardnets ? Or must land every now and then on a place with specific infra ?

**Comment by cryptomancer-actual** [comment]
Totally... a "golem familiar" or mount. Feel like designing it?

You could use the same format as the Familiars on page 260 of Cryptomancer.

**Comment by tromand** [comment]
I'l try something !

**Comment by tromand** [comment]
First Draft 

**GolemPhid :** 

A GolemPhid is a magically enhanced Gigaphid carrying a bit of the dwarf technology used in a Golem that can bridge shardnets. 
Because of the lack of infrastructure, they are not equivalent to a Golem:  they can only bridge 4 Shardnets, do not log accesses nor communications, and need to land to at least one rest period every seven period to maintenance in a safe-house containing at least a cryptoadmin and a ritual chamber. In any other way, they work like a Golem.

**Comment by tromand** [comment]
feel free to correct the wording to match with the rest of the Cryptomencer rules-book :)

**Comment by cryptomancer-actual** [comment]
Awesome, thanks! I'll play with it a little bit. Nice thing about a giant insect is that it has many legs, and probably holds a shard in each leg. Also, I love the maintenance window. Maybe that's the balancing feature... there is a certain % chance that the gigaphid might be resting, which can created an unplanned outage.

**Comment by tromand** [comment]
Yes and that also mean that during the "rest" time, the gigaphid is static (it can be localized or attacked)

**Comment by BakuDreamer** [comment]
Do gigaphids have wings ?  ( never actually says )  Or maybe they're like
giant dragon-caterpillars.  They sort of fly - ungulate like Flacor the
luck dragon.  ?    Giant caterpillars will eat an acre of rainforest every
day for sure.




​
​

**Comment by chadsansing** [comment]
Happy Day 2 of the Global Sprint!

Let us know how to help :)

cc @cryptomancer-actual, @tromand, @anterobot, @gael-ian, @Salindurthas, @DMRaven, @dexterduck, @Masterwolf2050, @ashgamedesign, @villain4hire, @Plotanical, @KadeMorton, @fabiocosta0305, @ksedivyhaley, @BrotherPhil

**Comment by tromand** [comment]
I'll be off for a few hours but will be back tonight (well, tonight for me ^^)

**Comment by chadsansing** [comment]
As the sprinting hour winds down on the east coast of the US, I want to thank all of you for your awesome contributions to this project.

We'll be curating and sharing throughout the summer. Stay tuned!

**Comment by fabiocosta0305** [comment]
How long the project will stay active?

**Comment by cryptomancer-actual** [comment]
@fabiocosta0305 I'll probably continue to harvest and incorporate ideas until July 1st. After that, will be spending a ton of time editing/curating/expanding user submissions.

**Comment by fabiocosta0305** [comment]
Oooh. Nice... I'll think some ideas and everything I have I'll put here

**Comment by BakuDreamer** [comment]
I thought this was done.  I still have some material, can I just put it in
the email here.

(  More Talents :  Nomenclator, Delimiter, and Stigmartyr )


Bokeh ( 'convoluted encryption', another kind of shadow magic )
Secret Twins ( cryptophasia as a spell ) ...
Umbrastion ( a cognitive firewall )

or should I just save it until next time ?

**Comment by cryptomancer-actual** [comment]
@BakuDreamer If the talents/spells have a clear learning nexus for people brand new to security/privacy and are based on concrete security/privacy concepts, feel free to drop them here. If they are more fictional or esoteric, you can put them in the /r/Cryptomancer sub-Reddit.

**Comment by BakuDreamer** [comment]
Bokeh ( blur ) is like 'multiple encryption'  Someone with the Bokeh talent
can 'bokeh' a message before it's encrypted, which changes it into asemic
text ( which can then be encrypted )  After it's sent and decrypted, it's
still bokeh'd and requires someone to un-bokeh it


​

**Comment by BakuDreamer** [comment]
​
Someone with ' Nomenclator ' has the ability to create fake true-name /
soul-key pairs and attach them to things.  But the names only last for 22.7
seconds before evaporating.

**Comment by BakuDreamer** [comment]
The Dancing Pig


​
A cantrip that puts a glamour-type charm in a Banner that makes you need to
roll a Willpower resistance check to not open it  ​
​

**Comment by BakuDreamer** [comment]
​

**Comment by fabiocosta0305** [comment]
@BakuDreamer looks like the kind of Cryptomancy that Brazilians would love! Hu3! :-D

**Comment by BakuDreamer** [comment]
Did the Risk Eaters pay for dinner ?   Something that might happen at the
masquerade-party.
​
